roles/nodepool/tasks/nodepool_v3.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Write logging config
+  template:
+    src: "etc/nodepool/logging.conf"
+    dest: "/etc/nodepool/{{ item }}_logging.conf"
+    owner: root
+    group: nodepool
+    mode: 0640
+  with_items:
+    - nodepool-builder
+    - nodepoold
+
+- name: Install systemd unit files
+  template:
+    src: "etc/systemd/system/{{ item }}.service"
+    dest: "/etc/systemd/system/{{ item }}.service"
+    owner: root
+    group: root
+    mode: 0644
+  with_items:
+    - nodepool-builder
+    - nodepoold
+  notify:
+    - Reload systemd units
+    - Restart nodepool
+
+- meta: flush_handlers
+
+- name: Enable and start systemd service
+  service:
+    enabled: yes
+    state: running
+    service: "{{ item }}"
+  with_items:
+    - nodepool-builder
+    - nodepoold
```

Comment:
It looks like the only thing that differs between nodepool_v2.yml  and nodepool_v3.yml is the services.  Could we have an internal '_nodepool_service_list' var that we use for with_items here, that is conditionally set to the correct services for the version of nodepol we're using? Then only have one nodepool.yml for both files